mmcv/ops/deform_conv.py

```diff
@@ -197,6 +197,13 @@ class DeformConv2d(nn.Module):
     `Deformable Convolutional Networks
     <https://arxiv.org/pdf/1703.06211.pdf>`_
 
+    Note:
+        The argument ``im2col_step`` was added in version 1.3.17, which means
+        number of samples processed by the ``im2col_cuda_kernel`` per call.
+        It enables users to define ``batch_size`` and ``im2col_step`` more
+        flexibly.Solved this `issue
```

Comment:
It enables users to define ``batch_size`` and ``im2col_step`` more flexibly and solved `issue mmcv#1440 <xxx>`_